crates/wasmcloud-host/src/capability/native_host.rs

```diff
@@ -58,12 +58,12 @@ impl Actor for NativeCapabilityHost {
             //warn!("Stopped a provider host that had no state. Something might be amiss, askew, or perchance awry");
             return;
         }
-        let mut state = self.state.as_mut().unwrap();
+        let state = self.state.as_mut().unwrap();
```

Comment:
Does this implicitly make a mutable reference, or does the `.as_mut()` not make a difference here

Reply:
yeah, it's an &mut